### DISCUSSION TITLE
Keybind Configs

### DISCUSSION THREAD

**Discussion by BellaRoka** [Malcolm-Q/LC-LateGameUpgrades#235] [Ideas]
Keybind configs would be super helpful, I use Middle Mouse Button for AdvancedCompany's headset item, however using it while carrying a shopping cart/wheelbarrow causes everything to drop. [LethalCompany InputUtils](https://thunderstore.io/c/lethal-company/p/Rune580/LethalCompany_InputUtils/) has a great implementation of the vanilla game's keybind menu.

**Comment by WhiteSpike** [comment]
3.2.0 release adds compatibility with [InputUtils](https://thunderstore.io/c/lethal-company/p/Rune580/LethalCompany_InputUtils/) so you are able to change it through its UI. If you don't have that mod installed, you can still change it through the configuration (though with a different format than previous versions)
